meetup.md

```diff
@@ -4,65 +4,63 @@ title: Meetup
-|            | Traits by Ivan Tham ([Slide](assets/2022/03/09/1-traits/slide.txt)) |
-| 2022-01-12 | Recording [Rust Lang Meetup - January 2022](https://www.youtube.com/watch?v=orh02-RucPo&list=PL85XCvVPmGQh3V0Pz-_xFm6VAUTR4aLUw&index=13) |
-|            | 3 Rust Myth - regarding error handling library, mutability and unsafe by Ivan Tham ([Slide](assets/2022/01/12/1-myth/slide.txt)) |
-| 2021-11-10 | Recording [Rust Lang Meetup - November 2021](https://www.youtube.com/watch?v=9oOEAGFTNdE&list=PL85XCvVPmGQh3V0Pz-_xFm6VAUTR4aLUw&index=13) |
-|            | Rust from a C/C++ perspective (more of a discussion) by Shawn Tan (sybreon) |
-| 2021-07-14 | Recording [Rust Lang Meetup - July 2021](https://www.youtube.com/watch?v=kJD-cJY2nb4&list=PL85XCvVPmGQh3V0Pz-_xFm6VAUTR4aLUw&index=11) |
-|            | Zero-cost abstarctions by Ivan Tham ([Slide](/assets/2021/07/14/1-zero-cost/slide.txt)) |
-| 2021-05-12 | Recording [Rust Lang Meetup - May 2021](https://www.youtube.com/watch?v=L3X-KJcGNg4&list=PL85XCvVPmGQh3V0Pz-_xFm6VAUTR4aLUw&index=10) |
-|            | PyOxidizer by Ivan Tham ([Slide](/assets/2021/05/12/1-pyoxidizer/slide.txt)) |
-|            | Implementing 3rd party trait for 3rd party type by Ivan Tham ([Slide](/assets/2021/05/12/2-integration/slide.txt)) |
-| 2021-03-10 | Recording [Rust Lang Meetup - March 2021](https://youtu.be/pH1JVMAh7Yw) |
-|            | Experience using diesel (and some bonus on a side project with nom and `inline-python!` pandas) by Ivan Tham |
-| 2020-01-08 | Recording [Rust Lang Meetup - January 2020](https://youtu.be/2jSfmZJxGZc) ([News](https://docs.google.com/presentation/d/1pezcgPIb485_ZR7U3yzo9KWOZ1j22m7y4diVmzgYUmc/edit?usp=sharing)) |
-|            | Road to async/.await, comparisons, examples by Ivan Tham ([Slides](/assets/2020/01/08/1-async/slide.txt)) |
-|            | DOM vs VDOM vs Rust-Dominator by Chong Tien Pow (remote from Penang) ([Slides](https://drive.google.com/file/d/1eDoGnKHo1wkfWAeL1GFGAS1wDEUuoWz8/view)) |
+| Date       | Info                                                                                                                                                                                                     |
+| ---------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 2023-04-12 | Recording [Rust Lang Meetup - April 2023](https://www.youtube.com/watch?v=hNponKesfiM&list=PL85XCvVPmGQh3V0Pz-_xFm6VAUTR4aLUw&index=19&t=3283s&ab_channel=RustMalaysia)                                  |
+| 2023-02-08 | Recording [Rust Lang Meetup - Febuary 2023](https://www.youtube.com/watch?v=rh1ZJXOuco8&list=PL85XCvVPmGQh3V0Pz-_xFm6VAUTR4aLUw&index=18)                                                                |
```

Comment:
Ah, this shouldn't be formatted, or otherwise every time we changed this this needs to be formatted as well. There is a reason why I purposely not format this, so it does not generate a big diff.

Reply:
By the way, we also need to put in the buzzcorner slides here.